src/TreeView/TreeView.stories.tsx

```diff
@@ -438,7 +438,7 @@ export const AsyncSuccess: Story = args => {
               <TreeView.DirectoryIcon />
             </TreeView.LeadingVisual>
             Directory with async items
-            <TreeView.SubTree state={state}>
+            <TreeView.SubTree state={state} count={args.count}>
```

Comment:
Could we create a new story (maybe called AsyncWithCount) that shows the skeleton loading items by default? I think it will make accessibility reviews easier if it's easy to jump to the loading spinner example and the skeleton items example.

Reply:
Sure thing!